r/R/dplyr.R

```diff
@@ -514,7 +515,7 @@ mutate.arrow_dplyr_query <- function(.data,
   if (!quo_is_null(.before) || !quo_is_null(.after)) {
     # TODO(ARROW-11701)
     return(abandon_ship(call, .data, '.before and .after arguments are not supported in Arrow'))
-  } else if (length(group_vars(.data)) > 0) {
+  } else if (length(dplyr::group_vars(.data)) > 0) {
```

Comment:
This might not be the absolutely right way to do this: at this point we will be using `group_vars.arrow_dplyr_query` which we do define, but R CMD check isn't able to catch that.

`dplyr::group_vars` is simply `UseMethod("group_vars")` so this should just work, but there might be a more approved solution for this case of referencing an s3 method without importing it directly

Reply:
Thanks, this is the correct way to do this IMO, dplyr is `Suggests`